### ISSUE TITLE
Validation loss equal to training loss

### ISSUE THREAD

**Issue by ulissesbcorrea** [CleanPegasus/TensorDash#18]
**Describe the bug**

Torchdash is plotting same values for training and validation data.

**Comment by CleanPegasus** [comment]
Hi! Thankyou for letting us know. Can you send us the code, so we can take a look?
